hazelcast-spring/src/main/resources/hazelcast-spring-3.9.xsd

```diff
@@ -2527,7 +2527,8 @@
             <xs:element name="eviction" type="eviction" minOccurs="0" maxOccurs="1"/>
         </xs:sequence>
         <xs:attribute name="in-memory-format" type="in-memory-format" use="optional" default="BINARY"/>
-        <xs:attribute name="invalidate-on-change" use="optional" type="xs:boolean" default="true"/>
+        <xs:attribute name="serialize-keys" use="optional" type="parameterized-boolean" default="true"/>
+        <xs:attribute name="invalidate-on-change" use="optional" type="parameterized-boolean" default="true"/>
```

Comment:
This PR also fixes the type of `invalidate-on-change` to `parameterized-boolean`.